doc/ics/yaml/gedling_gov_uk.yaml

```diff
@@ -0,0 +1,21 @@
+title: Gedling Borough Council (unofficial)
+url: https://github.com/jamesmacwhite/gedling-borough-council-bin-calendars
+howto: |
+   - Gedling Borough Council does not provide bin collections in the iCal calendar format directly.
+   - The iCal calendar files have been generated from the official printed calendars and hosted on GitHub for use.
+   - Go to: https://apps.gedling.gov.uk/refuse/search.aspx and enter your street name to find your bin day/garden waste collection schedule.
+   - Find the matching .ics file from within the ical folder from the unofficial repository.
```

Comment:
People not familiar with GitHub often struggle to download files from GitHub: I'd add a not that you have to click on the file and copy the link by clicking `copy raw file` to then use this as URL parameter

Reply:
Thanks. I've added further guidance/information.